all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -12,52 +12,63 @@
 # the License.
 #
 
-# This template uses Jaeger with in-memory storage with limited functionality
-# Do not use this in production environment!
-#
-# oc create -f jaeger-all-in-one-template.yml
-# oc process -f jaeger-all-in-one-template.yml | oc create -f -
-# oc delete all,template -l jaeger-infra
 parameters:
-- description: The name of the Jaeger
-  displayName: Jaeger service name
-  name: JAEGER_NAME
+- description: The name of the Jaeger Service.
+  displayName: Jaeger Service Name
+  name: JAEGER_SERVICE_NAME
+  required: true
+  value: jaeger
+- description: The namespace where to install the Jaeger instance
+  displayName: Namespace
+  name: NAMESPACE
```

Comment:
This is actually not required at all. It's there because we wanted to align it with the production template. And the production template has it set to `jaeger-infra` because it's actually a good idea to have a namespace dedicated to Jaeger and it should be our recommendation.